src/lib/TranscriptEditor/MediaPlayer/PlayerControls.module.css

```diff
@@ -20,24 +24,35 @@
   padding: 0.5em;
   border: 0;
   color: white;
+  background: color-darkest-grey;
   font-size: 1em;
   cursor: pointer;
+  /* border-color: black;
+  border-width: 0.1rem;
+  border-style: solid; */
+
+  margin-right: 0.3rem;
+  margin-top: 0.3rem;
 }
 
 .playBackRate{
+  height: 48px;
```

Comment:
I don't think this does anything here?

It's good to make sure each rule is doing what you think it's doing, or it'll turn into a headache afterwards 😱

Reply:
ok, well spotted, removed